packages/core/src/web/app/actions/dialog-caller.tsx

```diff
@@ -679,6 +680,24 @@ export default {
       />,
     );
   },
+  showStyleSelectionPanel: (onSelect: (style: string) => void, currentStyle: string): void => {
+    const id = 'style-selection-panel';
+
+    if (isIdExist(id)) {
+      return;
+    }
+
+    addDialogComponent(
+      id,
+      <StyleSelectionPanel
+        currentStyle={currentStyle}
+        onClose={() => popDialogById(id)}
+        onSelect={(style) => {
+          onSelect(style);
+        }}
+      />,
+    );
+  },
```

Comment:
nitpick:
In my opinion, I will recommend to put showStyleSelectionPanel in `AiGenerate/utils` instead import `StyleSelectionPanel` in `dialog-caller` to avoid circular import (in case `StyleSelectionPanel` also imported `dialog-caller` to call another dialog)